project-base/src/Model/Article/ArticleDataFactory.php

```diff
@@ -20,4 +21,15 @@ protected function createInstance(): BaseArticleData
     {
         return new ArticleData();
     }
+
+    /**
+     * @param \App\Model\Article\ArticleData $articleData
+     * @param \App\Model\Article\Article $article
+     */
+    protected function fillFromArticle(BaseArticleData $articleData, Article $article): void
+    {
+        parent::fillFromArticle($articleData, $article);
+
+        $articleData->createdAt = $article->getCreatedAt();
+    }
```

Comment:
You should fix this in framework class

Reply:
No, here is right place. Attribute createdAt in article would be added in the base project as an extension example, so having half the implementation in the framework and half in projet-bese is not possible.